changelog.d/8326.misc

```diff
@@ -0,0 +1 @@
+Moves every instance of `__metaclass__ = Y` to be `class X(metaclass=Y)`-based.
```

Comment:
Can you update this to be a bit more descriptive, maybe:

> Update the use of `metaclass` to be compatible with Python 3.

Not sure if that really makes sense or not though.

Also feel free to add "Contributed by <your name>." At the end if you'd like!

Reply:
Ah, right, I wanted to be a tad more technical in my wording, but this makes more sense, thanks!

Reply:
It's tough, usually the people reading these are not developers (not people who necessarily know Python well), so we try not to make them too deep in the weeds.

Then again, `misc` changes tend to do that. 👍